README.md

```diff
@@ -22,20 +20,158 @@ Or install it yourself as:
+  # Override the before and after hooks if you want to
+  # (See 'Controller before/after hooks')
+  #
+  # private
+  # def before_openstax_exception_rescue(exception_proxy)
+  #   # noop
+  # end
+  #
+  # def after_openstax_exception_rescue(exception_proxy)
+  #   respond_to do |f|
+  #     f.xml { render text: "I respond strangely to the XML format!",
+  #                    status: exception_proxy.status }
+  #   end
+  # end
+end
+```
+
+## Configuration
+
+This configuration, which is placed in `./config/initializers/rescue_from.rb` by the install generator, shows the defaults:
+
+```ruby
+OpenStax::RescueFrom.configure do |config|
+  config.raise_exceptions = ENV['RAISE_EXCEPTIONS'] || Rails.application.config.consider_all_requests_local
+
+  config.app_name = ENV['APP_NAME'] || 'Tutor'
+  config.app_env = ENV['APP_ENV'] || 'DEV'
+
+  config.notifier = ExceptionNotifier
+
+  config.html_error_template_path = 'errors/any'
+  config.html_error_template_layout_name = 'application'
+
+  config.email_prefix = "[#{app_name}] (#{app_env}) "
+  config.sender_address = ENV['EXCEPTION_SENDER'] || %{"OpenStax Tutor" <noreply@openstax.org>}
+  config.exception_recipients = ENV['EXCEPTION_RECIPIENTS'] || %w{tutor-notifications@openstax.org}
+end
+```
+
+## Registering exceptions
+
+In `./config/initializers/rescue_from.rb` it is recommended you register your exceptions
+
+Note that any unregistered exceptions rescued during run-time will be registered with the default options. See below.
+
+```ruby
+require 'openstax_rescue_from'
+
+OpenStax::RescueFrom.configure do |c|
+  # c.app_name ...
+end
+
+# OpenStax::RescueFrom#register_exception default options:
+#
+# { notify: true,
+#   status: :internal_server_error,
+#   extras: ->(exception) { {} } }
+#
+# Any unregistered exceptions rescued during run-time
+# will be registered during rescue with the options above
+
+OpenStax::RescueFrom.register_exception(SecurityTransgression,
+                                        notify: false,
+                                        status: :forbidden)
+
+OpenStax::RescueFrom.register_exception(ActiveRecord::NotFound,
+                                        notify: false,
+                                        status: :not_found)
+
+OpenStax::RescueFrom.register_exception(OAuth2::Error,
+                                        extras: ->(exception) {
+                                          { headers: exception.response.headers,
+                                            status: exception.response.status,
+                                            body: exception.response.body }
+                                        })
+
+OpenStax::RescueFrom.translate_status_codes({
+  forbidden: 'You are not allowed to access this.',
+  not_found: 'We couldn't find what you asked for.',
+})
+#
+# Default:
+#   internal_server_error: "Sorry, #{OpenStax::RescueFrom.configuration.app_name} had some unexpected trouble with your request."
+```
+
+## Override the views
```

Comment:
Before talking about overriding the views, can you talk about that there are views for errors?